scheduler/util.go

```diff
@@ -242,3 +296,114 @@ func tasksUpdated(a, b *structs.TaskGroup) bool {
 	}
 	return false
 }
+
+// setStatus is used to update the status of the evaluation
+func setStatus(logger *log.Logger, planner Planner, eval, nextEval *structs.Evaluation, status, desc string) error {
```

Comment:
Can you add unit tests for all 3 of these now that they are outside a specific scheduler? This will help catch future regressions